base/linalg/triangular.jl

```diff
@@ -1867,7 +1867,16 @@ function logm{T<:Union{Float64,Complex{Float64}}}(A0::UpperTriangular{T})
 end
 logm(A::LowerTriangular) = logm(A.').'
 
-function sqrtm{T}(A::UpperTriangular{T})
+function floop(x,R,i::Int,j::Int)
+    r = x
```

Comment:
This is not type stable if `A[i,j]` and `R[i,j]` are not the same type.   A simple fix is `r = x + zero(eltype(R))`